### ISSUE TITLE
New release needed for npm

### ISSUE THREAD

**Issue by JustAnotherGitHubUserYouDontKnow** [ENT8R/package-age#1]
I couldn't understand why, when installed via npm, I was getting an exception thrown on use that I could not correspond to the source code here.

Then I noticed the trivially locally resolved error (```options``` was unguarded for ```undefined``` on lines 65, 68 and 71 of my local index.js, but the function that provides this argument is called on line 222 without ```options``` defined) is not there on this github version, because a release is needed to push the two most recent commits into the npm registry.

In the meantime, for future googlers, my workaround was to locally change lines 65-73 of /usr/local/lib/node_modules/package-age/index.js (of the old 0.2.0 release) as follows:

```
        if (commander.dev || commander.all || options.dev || options.all) {
          result.devDependencies = await p(pkg.devDependencies, Types.DEV);
        }
        if (commander.peer || commander.all || options.peer || options.all) {
          result.peerDependencies = await p(pkg.peerDependencies, Types.PEER);
        }
        if (commander.bundled || commander.all || options.bundled || options.all) {
          result.bundledDependencies = await p(pkg.bundledDependencies, Types.BUNDLED);
        }
```
=>
```
        if (commander.dev || commander.all || (options && (options.dev || options.all))) {
          result.devDependencies = await p(pkg.devDependencies, Types.DEV);
        }
        if (commander.peer || commander.all || (options && (options.peer || options.all))) {
          result.peerDependencies = await p(pkg.peerDependencies, Types.PEER);
        }
        if (commander.bundled || commander.all || (options && (options.bundled || options.all))) {
          result.bundledDependencies = await p(pkg.bundledDependencies, Types.BUNDLED);
        }
```

Then it works great.

Of course, I could have pulled from git rather than npm, too...

In any case, please do a new release to get npm up to date with your latest commits! 😁

**Comment by ENT8R** [comment]
Thank you for bringing this to my attention 👍. I published the latest version as [`v0.3.0` on npm](https://www.npmjs.com/package/package-age) right now.

**Comment by JustAnotherGitHubUserYouDontKnow** [comment]
No worries, I've now upgraded to the most recent release and all works perfectly.

Thanks! 😊
